.travis.yml

```diff
@@ -12,11 +11,13 @@ env:
   - VARNISH_VERSION=4.0
 
 matrix:
+  allow_failures:
+    # HHVM support for CURLOPT_PROTOCOLS was added recently but not yet
```

Comment:
Okay, Guzzle 5 solves the `CURLOPT_PROTOCOLS` issue, but introduces [another one](https://travis-ci.org/FriendsOfSymfony/FOSHttpCache/jobs/66920349). Any clue?